test/utils/Spy.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const { assert } = require('chai');
+
+/**
+ * This class includes the utitity functions to assert spy data.
```

Comment:
Not a good description. Also, listing methods as a class doc quickly goes out-of-sync.